### PR TITLE
K2 drn touchup

### PR DESCRIPTION
**Pull request by JeffLCoughlin**
Made some minor-ish changes to the DRN that I wrote down as I did C2 and thought about future w/ global reprocessing:

-- Moved the reference to the Data Release Number from the end of each campaign to a bullet under the 'At a Glance' section. At the end of closeout, all data will have only one DRN number with live data at MAST. The exception is the data currently there is being archived as the final processing is delivered --- the data release notes (the ones for C0--C14 that currently exist) will be archived in the new archived data releaes note page with the old DRN number.  So on the main data release note page there is no need to have multiple DRN numbers --- just the number and information that corresponds to the data available at MAST.  To be clear, all the Data Release numbers are still on the pipeline release note page.

-- I went through Pipeline Release Notes page and updated 13 and 16 for C2 and C0, which were never delivered, to reflect that.

-- Added a h2 header called 'Data Quality and Processing Notes' to each campaign to differentiate notes about the data itself the existing 'Features and Events', which describes unique features of the campaign. I think helps differetiate between what is unique to each campain and what is unique about the data files at the archive. Going forward for reprocessed notes we should pretty much only have to update the 'Data Quality and Processing Notes' section.

- Minor consistency changes looking at all the the campaigns.

### CODE REVIEW

**Comment by JeffLCoughlin** [comment]
Taking the approach of charging ahead and we can update later on master if anyone spots typos or has objections/improvements.

File: content/pages/k2-observing/k2-data-release-notes.md [hunk 1]
```diff
@@ -243,16 +240,16 @@ The following is the data release history for C16. Follow the link for informati
...
+<li> Dec:  -20.0792397 degrees</li>
+<li> Roll:  166.7780778 degrees</li>
 </ul>
 
 <b><i>Targets</i></b>
 <ul>
 <li>  35,150 long cadence (LC) targets, including 3,485 galaxy targets.</li>
 <li>  118 short cadence (SC) targets.</li>
-<li>  38 moving objects were tiled with LC custom strip apertures. 13 bright stars were assigned 24-pixel diameter LC disk apertures to capture the point spread function wings. See the <a href="images/release-notes/c15/kplr2017355085300_c15_caf.csv">csv file that maps</a> the custom aperture number to the target name to find the apertures for a specific target.</li>
+<li>  38 moving objects we degreesre tiled with LC custom strip apertures. 13 bright stars were assigned 24-pixel diameter LC disk apertures to capture the point spread function wings. See the <a href="images/release-notes/c15/kplr2017355085300_c15_caf.csv">csv file that maps</a> the custom aperture number to the target name to find the apertures for a specific target.</li>
```
Comment (barentsen) [inline content/pages/k2-observing/k2-data-release-notes.md]: This looks like a typo: `we degreesre` =)

File: content/pages/k2-observing/k2-data-release-notes.md [hunk 2]
```diff
@@ -191,7 +195,7 @@ The pipeline CDPP 12th magnitude noise benchmark for C16 is
 the lowest seen since C6. It is comparable to that seen in early campaigns with similar
 star density (C6, C8, C10), but is well below that seen in C12, also with similar star density.
 While we do not have a definitive cause the for the improved precision, it is likely due to a
-combination of the relatively low star density, the return to more stable pointing (compared to recent
+combination of the relat degreesively low star density, the return to more stable pointing (compared to recent
```
Comment (barentsen) [inline content/pages/k2-observing/k2-data-release-notes.md]: Typo: `relat degreesively low star`

**Comment by barentsen** [comment]
@JeffLCoughlin Note the two small typos above (looks like a search-and-replace accident).  Otherwise :+1:.

**Comment by JeffLCoughlin** [comment]
Thx!  I need to disable my center mouse wheel from acting as a paste button....

Fixed with push to master.

**Pull request merged**